pkg/controller/storagecluster/storagecluster.go

```diff
@@ -1285,6 +1285,9 @@ func (c *Controller) getDefaultMaxStorageNodesPerZone(
 		return 0, err
 	}
 	numZones := len(zoneMap)
+	if numZones == 0 {
```

Comment:
nit: small remark -- could we change this info `if numZones =< 0 {` ?

When I see `if ==` checks like this, I always try to generalize and see if _"we can cover more ground, without paying more"_ (so, won't cost us anything to check `if =<` instead -- just in very unlikely (impossible?) case that `len()` function returns a negative value).

Reply:
Thanks Zoran for helping me out. I can always learn from your comments!